benchmarks/nemo/nemolite2d/psykal/Makefile_gen

```diff
@@ -80,17 +72,15 @@ nemolite2dxilinx: ${KERNELS} allkernels.xclbin psy.o alg.o
 
 %.o: %.mod
 
-# Xilinx Vitis OpenCL kernel compilation
-PLATFORM = xilinx_u200_xdma_201830_2
-EXECUTION_TARGET = sw_emu # Options: sw_emu | hw_emu | hw
+# OpenCL Ahead-of-time kernel compilation
```

Comment:
Although the compiler and flags are now variables, presumably the suffixes of the input and output files are still Xilinx specific? Please add a comment.

Reply:
Yes, although I suspect the suffixes are just a convention and will work regardless of the name. I added a comment.